tests/dummy/app/templates/index.hbs

```diff
@@ -22,6 +22,76 @@
   include to <code>app/styles/app.scss</code>.
 </p>
 
+<h3>Configuration</h3>
+
+<table class="uk-table uk-table-divider">
+  <thead>
+    <tr>
+      <th>Option</th>
+      <th>Type</th>
+      <th>Default</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>importUIkitCSS</code></td>
+      <td><code>Boolean</code></td>
+      <td><code>false</code></td>
+      <td>Whether to import the CSS of UIkit</td>
+    </tr>
+    <tr>
+      <td><code>importUIkitJS</code></td>
+      <td><code>Boolean</code></td>
+      <td><code>false</code></td>
+      <td>Whether to import the JS of UIkit</td>
+    </tr>
+    <tr>
+      <td><code>importUIkitAssets</code></td>
+      <td><code>Boolean</code></td>
+      <td><code>false</code></td>
+      <td>Whether to import the assets of UIkit</td>
+    </tr>
+    <tr>
+      <td><code>importUIkitIcons</code></td>
+      <td><code>Boolean</code></td>
+      <td><code>false</code></td>
+      <td>Whether to import the icons of UIkit</td>
+    </tr>
+    <tr>
+      <td><code>useIcons</code></td>
+      <td><code>Boolean</code></td>
+      <td><code>true</code></td>
+      <td>Whether to use UIkits icons</td>
+    </tr>
+    <tr>
+      <td><code>whitelist</code></td>
+      <td><code>String[]</code></td>
+      <td><code>[]</code></td>
+      <td>
+        A list of included components. Only components in this list will be
+        included in your build. You should never use <code>whitelist</code>
+        and <code>blacklist</code>!
```

Comment:
If you say that the two lists should not be used at the same time, i'd expect an error to be thrown in that case. I think right now the blacklist takes precedence.

Reply:
That should definitely throw an error!

Reply:
Or at least a warning